1.9/usage/pods/examples.md

```diff
@@ -0,0 +1,335 @@
+  "environment": null,
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": {
+        "command": {
+          "shell": "sleep 1000"
+        }
+      },
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32,
+        "disk": 0,
+        "gpus": 0
+      },
+      "endpoints": [],
+      "image": null,
+      "environment": null,
+      "user": null,
+      "healthCheck": null,
+      "volumeMounts": [],
+      "artifacts": [],
+      "labels": null,
+      "lifecycle": null
+    },
+    {
+      "name": "sleep2",
+      "exec": {
+        "command": {
+          "shell": "sleep 1000"
+        }
+      },
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32,
+        "disk": 0,
+        "gpus": 0
+      },
+      "endpoints": [],
+      "image": null,
+      "environment": null,
+      "user": null,
+      "healthCheck": null,
+      "volumeMounts": [],
+      "artifacts": [],
+      "labels": null,
+      "lifecycle": null
+    },
+    {
+      "name": "sleep3",
+      "exec": {
+        "command": {
+          "shell": "sleep 1000"
+        }
+      },
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32,
+        "disk": 0,
+        "gpus": 0
+      },
+      "endpoints": [],
+      "image": null,
+      "environment": null,
+      "user": null,
+      "healthCheck": null,
+      "volumeMounts": [],
+      "artifacts": [],
+      "labels": null,
+      "lifecycle": null
+    }
+  ],
+  "secrets": null,
+  "volumes": [],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ],
+  "scaling": {
+    "kind": "fixed",
+    "instances": 10,
+    "maxInstances": null
+  },
+  "scheduling": {
+    "backoff": {
+      "backoff": 1,
+      "backoffFactor": 1.15,
+      "maxLaunchDelay": 3600
+    },
+    "upgrade": {
+      "minimumHealthCapacity": 1,
+      "maximumOverCapacity": 1
+    },
+    "placement": {
+      "constraints": [],
+      "acceptedResourceRoles": []
+    }
+  }
+}
+```
+
+# A Pod that Uses Ephemeral Volumes
+
+```json
+{
+  "id": "/with-ephemeral-vol",
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "ct1",
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "exec": { "command": { "shell": "while true; do echo the current time is $(date) > ./jdef-v1/clock; sleep 1; done" } },
+      "volumeMounts": [
+        {
+          "name": "v1",
+          "mountPath": "jdef-v1"
+        }
+      ]
+    },
+    {
+      "name": "ct2",
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "exec": { "command": { "shell": "while true; do cat ./etc/clock; sleep 1; done" } },
+      "volumeMounts": [
+        {
+          "name": "v1",
+          "mountPath": "etc"
+        }
+      ]
+    }
+  ],
+  "networks": [
+    { "mode": "host" }
+  ],
+  "volumes": [
+    { "name": "v1" }
+  ]
+}
+```
+
+## IP-per-Pod Networking
+
+This pod declares a virtual (user) network named `my-virtual-network-name`.
```

Comment:
Consistently name the examples, e.g. `The following pod definition specifies...`